spec/factories/video.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :video do
+    title { "MyVideo" }
+    video_url { "www.example.com" }
```

Comment:
Similar to the other branch, can we make this a youtube url?

Reply:
Agreed. I will update this one.